### PR TITLE
add blog post about renaming master -> main

### PR DESCRIPTION
**Pull request by polandll**

### CODE REVIEW

File: _posts/2021-12-14-announce-github-naming-change.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: "Announcing Github renaming change"
```
Comment (dougwettlaufer) [inline _posts/2021-12-14-announce-github-naming-change.md]: Should we specify this is the branch being renamed? Just to any confusion for someone glancing at the title but not reading the full blog? Not that I've ever done something like that 🙂

File: _posts/2021-12-14-announce-github-naming-change.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+date:   2021-12-14 00:00:00 -0600
+synopsis: "Github is changing master to main, and we are, too."
+author_info:
+  name:
+    first: "Lorina"
+    last: "Poland"
+  picture: "assets/images/stargate-profile.png"
+---
+
+In October 2020 , all new source code repositories created on GitHub will be named 'main' instead of `master` as part of the company's effort to remove unnecessary references to slavery and replace them with more inclusive terms. This "fix forward first" approach is the first step in making systemic changes to the naming conventions.
```
Comment (mlr) [inline _posts/2021-12-14-announce-github-naming-change.md]: extra space before the comma after 2020

Comment (mlr) [reply _posts/2021-12-14-announce-github-naming-change.md]: should the `main` and `master` use backticks or single quotes consistently with each other?

File: _posts/2021-12-14-announce-github-naming-change.md (hunk 1 above)
Comment (polandll) [reply _posts/2021-12-14-announce-github-naming-change.md]: Changed title: "Announcing Github branch renaming change"

File: _posts/2021-12-14-announce-github-naming-change.md (hunk 2 above)
Comment (polandll) [reply _posts/2021-12-14-announce-github-naming-change.md]: Done, and done.

**Review by mlr — APPROVED** [review]

**Pull request merged**